neo/app/assets/javascripts/lib/adf-structures-base-0.1.1/.jshintrc

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Oh you mean this file, this must generated by the framework, I will update gitignore for that